### PR TITLE
Changes to Worldpay go live content driven by Gwent Police

### PR DESCRIPTION
**Pull request by jonathanglassman**
https://payments-platform.atlassian.net/browse/PP-7546

Implement fixes to improve Worldpay go live docs:

- Make it clear which screen you are in for each step
- "Refused by bank" - users unable to check the checkbox in merchant channel

### CODE REVIEW

File: source/switching_to_live/set_up_a_live_worldpay_account/index.html.md.erb [hunk 1]
```diff
@@ -37,13 +37,13 @@ If __Test Mode__ is at the bottom of the left-hand menu, switch to __Production
 
 3. In your Worldpay account, select __Account__ and the __Profile__ tab, then your merchant code if you're asked for one.
 
-4. Copy the __Merchant Code__ value from the __Identification__ section of your Worldpay account into the __Merchant code__ field in your GOV.UK Pay account.
+4. In your Worldpay account, copy the __Merchant Code__ value from the __Identification__ section. Pastes this value into the __Merchant code__ field in your GOV.UK Pay account.
```
Comment (stephencdaly) [inline source/switching_to_live/set_up_a_live_worldpay_account/index.html.md.erb:40]: Typo, "Pastes" -> "Paste"

**Review by stephencdaly — CHANGES_REQUESTED** [review]
Looks good, changes made should clarify points that confused Gwent Police.

Just one typo

File: source/switching_to_live/set_up_a_live_worldpay_account/index.html.md.erb (hunk 1 above)
Comment (jonathanglassman) [reply source/switching_to_live/set_up_a_live_worldpay_account/index.html.md.erb:40]: Whoops. Should not have done this in a hurry. Well-spotted, will change.

Comment (jonathanglassman) [reply source/switching_to_live/set_up_a_live_worldpay_account/index.html.md.erb]: Done @stephencdaly

**Review by stephencdaly — APPROVED** [review]

**Pull request merged**